cluster/addons/addon-manager/kube-addon-update.sh

```diff
@@ -491,6 +491,7 @@ function update-addons() {
     reconcile-objects ${addon_path} PersistentVolume "" &
     reconcile-objects ${addon_path} PersistentVolumeClaim "" &
     reconcile-objects ${addon_path} ConfigMap "" &
+    reconcile-objects ${addon_path} StorageClass "" &
```

Comment:
We need a way to reconcile the existence of a storage class without reconciling the selection of that class as teh default (an annotation).  This is similar to the replica count.

@mikedanes and @MrHohn because you're ripping this all up